install_config/master_node_configuration.adoc

```diff
@@ -1743,10 +1743,12 @@ To launch a node server using a node configuration file:
 $ openshift start node --config=/openshift.local.config/node-<node_hostname>/node-config.yaml
 ----
 
-[[master-node-config-logging-levels]]
-== Configuring Logging Levels
+[[master-node-view-logs]]
+== Viewing Master and Node Logs
+
+{product-title} uses the `systemd-journald.service` to collect log messages for debugging the masters and nodes.
 
-{product-title} uses the `systemd-journald.service` to collect log messages for debugging, using five log message severities. The logging levels are based on Kubernetes logging conventions, as follows:
+The logging using five log message severities based on Kubernetes logging conventions, as follows:
```

Comment:
> The logging **using** ... **based** on Kubernetes logging conventions

The logging **uses** ... based on Kubernetes logging conventions
Or
The logging using ... **is** based on Kubernetes logging conventions